File: ws-eclipse/projetos_udemy/curso_udemy/src/crescente.java
import java.util.Locale;
import java.util.Scanner;

public class crescente {
public static void crescente(String[] args) {
	
 Locale.setDefault(Locale.US);
 Scanner sc = new Scanner(System.in);

 int n1, n2;
 
 System.out.println("Digite o primeiro valor: ");
 n1 = sc.next();
 
 System.out.println("Digite o segundo valor : ");
 n2 = sc.next();
 
 while(n1 > n2) {
	 System.out.println("DECRESCENTE");
 }
 
 while(n2 > n1) {
	 System.out.println("CRESCENTE");
 }

}
}

File: ws-vs/Primeiro/CRESCENTE.cs
using System.Globalization;

namespace Programa {
    class Program {
        static void Main(string[] args) {

            int valor1, valor2;

            Console.WriteLine("Digite o primeiro valor : ");
            valor1 = int.Parse(Console.ReadLine());

            Console.WriteLine("Digite o segundo valor : ");
            valor2 = int.Parse(Console.ReadLine());      

            if(valor1 < valor2){
                Console.WriteLine("CRESCENTE");
                else{
                    Console.WriteLine("DECRESCENTE");
                }
            }  
        }
    }
}